modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
+* The NIC must be tagged to all relevant VLANs.
+** For example: `bridge vlan add dev bond0 vid 1-4095 master`
+
+.Procedure
+
+. Create a new file for the NetworkAttachmentDefinition in any local directory.
+The file must have the following contents, modified to match your
+configuration:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: a-bridge-network
```

Comment:
@ousleyp can you add here
```
  annotations:
    k8s.v1.cni.cncf.io/resourceName: bridge-cni.network.kubevirt.io/br0 <1>
```
and a note

<1> if this annotation is added, VMs would be scheduled only on hosts that have this bridge available.

We can mention that this is implemented by the bridge-marker, but I'd rather we do not include @mmirecki's text about its installation as it should be working out of the box.

@ousleyp I don't manage to directly reply to your question https://github.com/openshift/openshift-docs/pull/14351#discussion_r281359810 (I never got the hang of github :-/ )
We can live without this text, but it would be great to have it included.